3 - PB/Documentazione interna/Verbali/24-03-06/24-03-06.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "06/03/24",
+  subTitle: "Meeting post colloquio con Committente",
+  docType: "verbale",
+  authors: (
+    "Zaccone Rosario",
+  ),
+  missingMembers: (
+    "Banzato Alessio",
+  ),
+  timeStart: "09:00",
+  timeEnd: "10:00",
+);
+
+= Ordine del giorno
+A seguito dell'incontro con il #cardin, il gruppo ha svolto un meeting interno riguardante:
+
+- Considerazioni scaturite dal colloquio con il #cardin ;
+- Pianificazione.
+
+== Considerazioni scaturite dal colloquio con il #cardin
+Il gruppo ha discusso i diversi argomenti trattati con il #cardin durante il meeting.
+=== Database
+Il gruppo ha deciso di scegliere la linea di progettazione che vede il database come parte del capitolato, e in particolare come data layer del software. Inoltre si è scelto di accedervi tramite backend, poiché farlo tramite browser porterebbe a numerosi problemi di sicurezza.
+La scelta è stata conseguita in seguito ai consigli del #cardin, che ha anche portato l'esempio del pattern CQRS come caso di design dove un database viene usato per sola lettura, come nel caso di WMS3. Il gruppo sceglie però di non implementare questo pattern, dato che nel suo schema è presente una parte dedicata alla scrittura dei dati che non è oggetto del capitolato.
+=== Classi anemiche
+Il gruppo ha deciso di proseguire con l'implementazione di classi che rappresentano le istanze del database.
+Il dubbio sull'implementarle o meno, poiché quest'ultime sono prive di comportamento specifico e risultano un semplice aggregato di attributi, è stato risolto grazie all'intervento del #cardin, che nel meeting ha spiegato al gruppo il significato di questo tipo di classi, chiamate _anemiche_, e ha asserito che la loro implementazione è sensata.
+=== Business Logic
+Il gruppo ha discusso sull'effettiva presenza della business logic in seguito al feedback ricevuto dal #cardin. La bl è stata individuata principalmente nella parte che interroga l’API REST esterna e negli oggetti che rappresentano le istanze del database. Si è discusso il modo in cui gli oggetti della bl devono essere convertiti in oggetti della presentation logic, e si è posta la norma di non utilizzare gli oggetti ritornati dalle API direttamente nella webapp.
```

Comment:
scrivere per esteso business logic e non solo bl, oppure riformulare in "Questa è stata individuata..." per evitare ripetizioni